win_bin/install/install-dcos-cli.ps1

```diff
@@ -83,11 +83,7 @@ if ($PYTHON_ARCHITECTURE -eq 64) {
   & $installation_path\Scripts\easy_install  "http://downloads.sourceforge.net/project/pywin32/pywin32/Build%20219/pywin32-219.win32-py$PYTHON_VERSION.exe" 2>&1 | out-null
 }
 
-if ($env:DCOS_CLI_VERSION) {
-  & $installation_path\Scripts\pip install --quiet "dcoscli==$env:DCOS_CLI_VERSION"
-} else {
-  & $installation_path\Scripts\pip install --quiet "dcoscli"
-}
+& $installation_path\Scripts\pip install --quiet "dcoscli<0.4.0"
```

Comment:
Yes but this will overwrite what we have already released, no? It will
overwrite the wrong file, no?

On Wed, Feb 24, 2016 at 1:17 PM tamarrow <notifications@github.com> wrote:

> In win_bin/install/install-dcos-cli.ps1
> <https://github.com/mesosphere/dcos-cli/pull/491#discussion_r54007729>:
>
> > @@ -83,11 +83,7 @@ if ($PYTHON_ARCHITECTURE -eq 64) {
> >    & $installation_path\Scripts\easy_install  "http://downloads.sourceforge.net/project/pywin32/pywin32/Build%20219/pywin32-219.win32-py$PYTHON_VERSION.exe" 2>&1 | out-null
> >  }
> >
> > -if ($env:DCOS_CLI_VERSION) {
> > -  & $installation_path\Scripts\pip install --quiet "dcoscli==$env:DCOS_CLI_VERSION"
> > -} else {
> > -  & $installation_path\Scripts\pip install --quiet "dcoscli"
> > -}
> > +& $installation_path\Scripts\pip install --quiet "dcoscli<0.4.0"
>
> this is for legacy cli. current master has two install scripts, 1 for
> legacy and 1 for current
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/dcos-cli/pull/491/files#r54007729>.
>
-- 
José Armando García Sancio
Mesosphere